project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java

```diff
@@ -33,7 +33,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Optional<VolunteeringOpportunity> opportunity =
         databaseWrapper.getVolunteeringOpportunityByOppportunityId(opportunityId);
 
-    if (opportunity.isEmpty()) {
+    if (!opportunity.isPresent()) {
```

Comment:
Let change this to opportunity.isPresent() and swap the blocks? I think it will be more straightforward for readers..

Reply:
Done